src/libs/Network/SequentialQueue.ts

```diff
@@ -71,7 +70,7 @@ function process() {
         .then((response) => {
             // A response might indicate that the queue should be paused. This happens when a gap in onyx updates is detected between the client and the server and
             // that gap needs resolved before the queue can continue.
-            if (response.shouldPauseQueue) {
+            if (response?.shouldPauseQueue) {
```

Comment:
Because of this change tests started to fail, turns out it uncovered a potential regression. I described it [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1696598950477539).

So for now this PR is kind of blocked @mollfpr @iwiznia